docs/src/content/callbacks.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Callbacks'
+---
+
+import Examples from 'components/examples'
+import Contents from 'components/table-of-contents'
+
+# Accepting callbacks
+
+A callback lets you receive messages from Notify to a URL you choose.
```

Comment:
`lets you receive messages from Notify` - this sounds like the messages/notifications themselves are sent to the URL.

```suggestion
You can configure a callback to receive message (sms/email) receipt updates and/or inbound sms messages.
```